src/components/renderer/form-new-request.vue

```diff
@@ -77,6 +77,13 @@ export default {
           data.meta.from -= 1;
           this.$refs.listProcess.data = data;
           this.$refs.listProcess.setPaginationData(data.meta);
+          const dataStart = {
```

Comment:
for this control the counter is not required

Reply:
As we agree with Fabio explaining this code, this counter is needed and is only applied to tasks and requests to obtain the number of rows returned in Api response to emit the event to father component.